### ISSUE TITLE
Some students can't access test exam

### ISSUE THREAD

**Issue by MaximilianAnzinger** [ls1intum/Artemis#9749]
### Describe the bug

Some of my students seemingly can't access a test exam.

### To Reproduce

1. Login as a student
2. Navigate to exams
3. Select test exam
4. Internal Server Error

### Expected behavior

All students of the course should be able to participate int he mock exam

### Screenshots

![image](https://github.com/user-attachments/assets/cdc931fb-d7a4-4ccf-b593-a18567060d54)

### Which version of Artemis are you seeing the problem on?

7.7.0

### What browsers are you seeing the problem on?

Chrome

**Comment by badkeyy** [comment]
Can't reproduce this with my test exam... Do you have a more precise description of the reproduction steps/does this happen always for any given student or not?

**Comment by Strohgelaender** [comment]
Can you check if two student exams for this one student somehow got created? I remember having a similar issue in https://github.com/ls1intum/Artemis/issues/6208

**Comment by MaximilianAnzinger** [comment]
I agree its a very very rare case. It happened to us on production. So if someone is willing to take on the ticket I can share the affected individual, exam, etc. .

**Comment by badkeyy** [comment]
@MaximilianAnzinger - I would love to have a look at it.

**Comment by MaximilianAnzinger** [comment]
@badkeyy feel free to hit me up tomorrow during the dev day 👍

**Comment by Jess-hub09** [comment]
I’m not able to reproduce this issue, neither on TS5 with test student 3 nor with my own account. I also didn’t receive any reports from students during the exam supervision about access problems. From my side everything seems to work as expected.
